modules/oe_theme_helper/webpack.config.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const fs = require('fs');
+const webpack = require('webpack');
+const { styles, builds } = require('@ckeditor/ckeditor5-dev-utils');
+const TerserPlugin = require('terser-webpack-plugin');
+
+function getDirectories(srcpath) {
```

Comment:
This function should have an annotation describing its purpose.

Reply:
This file is the same as the webpack.config.js present in core, with modifications provided by the ckeditor5_dev module (maintained by core people). I would not introduce changes to it, for ease of updates in the future and consistency. This is valid for the points above too

Reply:
For reference: https://git.drupalcode.org/project/ckeditor5_dev/-/blob/1.0.x/ckeditor5_plugin_starter_template/webpack.config.js